include/curl/curl.h

```diff
@@ -743,6 +743,15 @@ typedef CURLcode (*curl_ssl_ctx_callback)(CURL *curl,    /* easy handle */
                                                           mbedtls_ssl_config */
                                           void *userptr);
 
+/* This is a return codes for the ssl distinguished name callback that, when returned
```

Comment:
Invalid grammar, use of `a` in plural `codes`

Reply:
Fixed.